config/initializers/simple_form_bulma.rb

```diff
@@ -543,56 +543,56 @@
     # end
 
 # rubocop:enable Layout/CommentIndentation
+end
```

Comment:
Everything below here in this file is just whitespace change; moved the class definitions below out of the block above.